src/Service/PaymentStatusService.php

```diff
@@ -91,8 +91,7 @@ public function getWithOrderId(string $orderId): array
 
         if (empty($paymentResponse)) {
             throw new MissingDataException(
-                'Payment response cannot be found for order number: ' .
-                $paymentResponse->getOrderTransaction()->getOrder()->getOrderNumber() . '!'
+                'Payment response cannot be found for order id: ' . $orderId . '!'
```

Comment:
I understand but we don't have an orderNumber at that moment of logging :(
I changed the log because if the $paymentResponse is empty therefore the ```$paymentResponse->getOrderTransaction()->getOrder()->getOrderNumber()``` is going to throw an exception